internal/utils/templates/initial_schemas/14.sql

```diff
@@ -25,6 +25,16 @@ CREATE SCHEMA IF NOT EXISTS _realtime;
 
 ALTER SCHEMA _realtime OWNER TO postgres;
 
+--
+-- Name: _analytics; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA IF NOT EXISTS _analytics;
+
+
+ALTER SCHEMA _analytics OWNER TO postgres;
+ALTER USER postgres WITH REPLICATION;
```

Comment:
The `postgres` user already has replication attribute by default

```bash
postgres=# \du postgres
                                                     List of roles
 Role name |                         Attributes                         |                  Member of
-----------+------------------------------------------------------------+----------------------------------------------
 postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {supabase_auth_admin,supabase_storage_admin}
```

Reply:
👍